### PR TITLE
Improvements to Script

### PR DESCRIPTION
**Pull request by ananion-cation**

### CODE REVIEW

File: newscript.go [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+
+func cleanup() error {	
+	filesToRemove := [...]string{linuxVersion, "NiChrome", "vboot_reference"}
+	fmt.Printf("-------- Removing problematic files %v\n", filesToRemove)
+	for _,file := range filesToRemove{
+		if _, err := os.Stat(file); err != nil {
+			continue		
+		} 
+		err := os.Remove(file)
```
Comment (rminnich) [inline newscript.go]: does this work on directories?

File: newscript.go [hunk 2]
```diff
@@ -0,0 +1,404 @@
...
+	err = cmd5.Run()
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// To get the newest Kernel from kernel.org
+func newestkernelGet() error {
+	jsonFile, err := http.Get("http://www.kernel.org/releases.json")
```
Comment (rminnich) [inline newscript.go]: slick! This is neat.

File: newscript.go [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+	if err != nil {
+		return err
+	}
+	defer jsonFile.Body.Close()
+	d, err := ioutil.ReadAll(jsonFile.Body)
```
Comment (rminnich) [inline newscript.go]: This is not as neat ;-)
you should use the encoding/json package to decode the json to a struct. The way you're doing this is fragile. We can get this later.

File: newscript.go [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+		fmt.Printf("didn't clone Nichrome")
+		return err
+	}
+	cmd9 := exec.Command("cp", "NiChrome/CONFIG", fmt.Sprintf("%s/.config", linuxVersion))
+	err = cmd9.Run()
+	if err != nil {
+		fmt.Println("3")
+		return err
+	}
+	cmd10 := exec.Command("cp", "/tmp/initramfs.linux_amd64.cpio", fmt.Sprintf("%s/", linuxVersion))
```
Comment (rminnich) [inline newscript.go]: or, if you want, do an os.Symlink here and save the time of the copy.

**Comment by rminnich** [comment]
looks like you've got a need to rebase or something?

**Comment by rminnich** [comment]
It doesn't work well to reopen a commit like this. It was merged. 
you can do something like this from your desktop

let's pretend you are working on the script branch again and you've done your pull --rebase

git push ananion-cation :script
NOTE the ':' -- this causes the remote branch at github to be removed.
git push ananion-cation
this makes what looks like a new script branch.

But don't reopen PRs like this once the commit has happened, it rarely ends well (unless you've figured a way to do this I don't know about ...)

File: newscript.go (hunk 1 above)
Comment (ananion-cation) [reply newscript.go]: Changed to RemoveAll in new version

**Pull request closed without merging**